config/webpack.config.js

```diff
@@ -118,8 +126,20 @@ var config = {
         NOTIFICATIONS_POLLING: process.env.NOTIFICATIONS_POLLING
       }
     }),
+    // Put chunk in vendor if
+    // node_module and used by bundle
+    // node_module and used by multiple entries
     new webpack.optimize.CommonsChunkPlugin({
-      name: 'vendor'
+      name: 'vendor',
+      minChunks: function(module, count) {
+        const rootIssuer = getModuleRootIssuer(module);
+
+        const isNodeModule = module.context && module.context.includes("node_modules");
+        const isUsedByBundle = rootIssuer.resource === bundle;
+        const isUsedByMultipleEntries = count > 1;
+
+        return isNodeModule && (isUsedByBundle || isUsedByMultipleEntries);
```

Comment:
Won't the `... || isUsedByMultipleEntries` part produce different results for production and for development envs?

Reply:
@tstrachota You right, Thanks!

@ohadlevy I still haven't found a good approach here, right now, my best solution is to create a list of node_modules that should get into the vendor.js.

Reply:
@ohadlevy When i think about it again, using a vendors list can cause problems with packages using other packages...